pom.xml

```diff
@@ -83,35 +109,40 @@
       <version>1.10</version>
     </dependency>
     <dependency>
-      <groupId>org.xerial</groupId>
-      <artifactId>sqlite-jdbc</artifactId>
-      <version>3.7.2</version>
-    </dependency>
-    <dependency>
-        <groupId>commons-configuration</groupId>
-        <artifactId>commons-configuration</artifactId>
-        <version>1.6</version>
+      <groupId>commons-configuration</groupId>
+      <artifactId>commons-configuration</artifactId>
+      <version>1.10</version>
     </dependency>
     <dependency>
         <groupId>org.twitter4j</groupId>
-        <artifactId>twitter4j-core</artifactId>
-        <version>3.0.3</version>
+        <artifactId>twitter4j-stream</artifactId>
+        <version>[4.0,)</version>
```

Comment:
It doesn't need to be hardcoded. I meant this has no upper bound. It should be [4.0,5) so it doesn't update beyond 4.x. Possibly even [4.0,4.1). We should define an upper bound here.